<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quad</class>
 <widget class="QWidget" name="quad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>325</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>325</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(117, 64, 238);
border-radius: 20px;
</string>
  </property>
  <widget class="QPushButton" name="pushButton_17">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 0px 0px 5px 5px;
font: 11pt &quot;Inter&quot;;

color: rgb(255, 255, 255);
}




</string>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
font: 700 19pt &quot;Inter&quot;;</string>
   </property>
   <property name="text">
    <string>Квадратное уравнение</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_quad_eq">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>19</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
font: 700 19pt &quot;Inter&quot;;</string>
   </property>
   <property name="text">
    <string>Ax²+ bx + c = 0</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLineEdit" name="b_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLineEdit" name="a_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLineEdit" name="c_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
font: 700 16pt &quot;Inter&quot;;</string>
   </property>
   <property name="text">
    <string>c:</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_7" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLineEdit" name="disc_Answer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>12</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>420</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
font: 700 19pt &quot;Inter&quot;;</string>
   </property>
   <property name="text">
    <string>Дискриминант:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>510</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
font: 700 19pt &quot;Inter&quot;;</string>
   </property>
   <property name="text">
    <string>Корни:</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_8" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>550</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLineEdit" name="x1_Answer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_9" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>610</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLineEdit" name="x2_Answer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>7</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
font: 700 16pt &quot;Inter&quot;;</string>
   </property>
   <property name="text">
    <string>X1:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>760</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 10px;
font: 10pt &quot;Inter&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
}


QPushButton:hover {
	background-color: rgb(45, 45, 45);
}



</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>760</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 10px;
font: 10pt &quot;Inter&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
}


QPushButton:hover {
	background-color: rgb(45, 45, 45);
}



</string>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
font: 700 16pt &quot;Inter&quot;;</string>
   </property>
   <property name="text">
    <string>b:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
font: 700 16pt &quot;Inter&quot;;</string>
   </property>
   <property name="text">
    <string>A:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calc_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 10px;
font: 10pt &quot;Inter&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
}


QPushButton:hover {
	background-color: rgb(45, 45, 45);
}



</string>
   </property>
   <property name="text">
    <string>Решить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(255, 255, 255);
font: 700 16pt &quot;Inter&quot;;</string>
   </property>
   <property name="text">
    <string>X2:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
